tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceFactoryTests.cs

```diff
@@ -75,7 +75,8 @@ private static IProjectVersionedValue<WorkspaceUpdate> EvaluationUpdate()
         ConfiguredProject configuredProject = ConfiguredProjectFactory.Create();
         IProjectSubscriptionUpdate evaluationRuleUpdate = IProjectSubscriptionUpdateFactory.CreateEmpty();
         IProjectSubscriptionUpdate sourceItemsUpdate = IProjectSubscriptionUpdateFactory.CreateEmpty();
-        var workspaceUpdate = WorkspaceUpdate.FromEvaluation((configuredProject, evaluationRuleUpdate, sourceItemsUpdate));
+        IProjectSnapshot projectSnapshot = IProjectSnapshot2Factory.Create();
+        var workspaceUpdate = WorkspaceUpdate.FromEvaluation((configuredProject, projectSnapshot, evaluationRuleUpdate, sourceItemsUpdate));
```

Comment:
*joke* Double parentheses!! WHAT DOES IT MEAN?!?! 🌈🌈

ValueTuples are weird. 🙃